infrastructure/modules/container-app/variables.tf

```diff
@@ -36,6 +35,17 @@ variable "fetch_secrets_from_app_key_vault" {
   nullable    = false
 }
 
+variable "fetch_secrets_from_infra_key_vault" {
```

Comment:
The infra key vault is created by bicep before terraform runs. So we could remove this variable and only use var.enable_auth instead